### PR TITLE
Fix method POST

### PR DESCRIPTION
**Pull request by sekot**

### CODE REVIEW

**Comment by sekot** [comment]
Maybe you need to check for AD login, I m not sure that is working, if not maybe add a little test for determinate if AD or LDAP

**Comment by Aidaho12** [comment]
Awesome! Did you test it?

File: app/templates/ajax/overview.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {% for service in service_status %}
+{{ service_status }}
```
Comment (Aidaho12) [inline app/templates/ajax/overview.html]: What for did you delete loop?

Comment (sekot) [reply app/templates/ajax/overview.html]: oh , my bad, i forget a debug print, that delete nothing

**Comment by sekot** [comment]
Not All.
login, settings, update/copy haproxy/keepalive file
I m testing your solution.
i want add too ".map" for choose backend in frontend, i can bypass with white list, but i do not like that.
And add the errors files ".http"
I think copy your white list script for the two feature

**Comment by Aidaho12** [comment]
>i want add too ".map" for choose backend in frontend, i can bypass with white list, but i do not like that.
And add the errors files ".http"
I think copy your white list script for the two feature

You talk about new features?

**Comment by sekot** [comment]
Yes
-------- Message d'origine --------De : Pavel Loginov <notifications@github.com> Date : 04/12/2019  19:14  (GMT+01:00) À : Aidaho12/haproxy-wi <haproxy-wi@noreply.github.com> Cc : Frédéric COELHO <frederic.coelho@gmail.com>, Author <author@noreply.github.com> Objet : Re: [Aidaho12/haproxy-wi] Fix method POST (#189) 
i want add too ".map" for choose backend in frontend, i can bypass with white list, but i do not like that.
And add the errors files ".http"
I think copy your white list script for the two feature

You talk about new features?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/Aidaho12/haproxy-wi/pull/189?email_source=notifications\u0026email_token=AAFKYELZWPYSZ5NAFOQ5B53QW7XQNA5CNFSM4JVIZF5KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEF57PZI#issuecomment-561772517",
"url": "https://github.com/Aidaho12/haproxy-wi/pull/189?email_source=notifications\u0026email_token=AAFKYELZWPYSZ5NAFOQ5B53QW7XQNA5CNFSM4JVIZF5KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEF57PZI#issuecomment-561772517",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Aidaho12** [comment]
I have thought about this, but I don't think that it is popular functions

**Comment by Aidaho12** [comment]
Thank you for commit )

**Pull request merged**